make/env.mk

```diff
@@ -17,6 +17,8 @@ CGO_ENABLED := 1
 # Update the arch to arm64 but only for Macs running on Apple Silicon (M1)
```

Comment:
nit: this comment isn't correct anymore, as it's updated for ARM Linux machines, too